Tutorials/get-the-tutorials.md

```diff
@@ -0,0 +1,32 @@
+---
+toc: true
+title: Getting the tutorials
+layout: tutorial
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Getting the tutorials
+## Python Dependencies
+Additional python and CAmkES dependencies are required to build [tutorials](ReworkedTutorials). To install you can run:
+```
+pip3 install --user aenum pyelftools camkes-deps
```

Comment:
Only `camkes-deps` is required, `aenum` and `pyelftools` are part of `camkes-deps` already.

```suggestion
pip3 install --user aenum pyelftools camkes-deps
```